docker/jukebox.Dockerfile

```diff
@@ -45,7 +47,8 @@ RUN [ "$(uname -m)" = "aarch64" ] && ARCH="arm64" || ARCH="$(uname -m)"; \
     rm -f libzmq.tar.gz;
 
 RUN export ZMQ_PREFIX=${PREFIX} && export ZMQ_DRAFT_API=1
-RUN pip install -v --no-binary pyzmq --pre pyzmq
+#RUN pip install -v --no-binary pyzmq --pre pyzmq
+RUN pip install -v --no-binary=:all: pyzmq --pre pyzmq
```

Comment:
Please use the version of `future3/develop` here. these changes have already been made to this branch.